next.clayui.com/content/docs/css/components/tabs.md

```diff
@@ -0,0 +1,634 @@
+---
+title: "Tabs"
+description: "Tabs organize similar content together into individual sections in the same page."
+---
+
+<div class="nav-toc">
+
+- [Classic](#classic)
+- [Modern](#modern)
+- [Variations](#variations)
+    - [Buttons](#buttons)
+	- [Justified](#justified)
```

Comment:
Should this be on the same line as buttons? We can use justified without buttons right?

Reply:
Same related issue of ghost spaces. Fixing!